bench/parser/bench_parser.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+"""
+This script benchmarks the xDSL parser by parsing all files in the
+given root directory.
+It then prints the total time taken to parse all files.
+Each file is first parsed by MLIR to check that it is valid, and in
+the generic form.
+Each file is also splitted according to `split-input-file` if it is
+present.
+"""
+
+import subprocess
+import cProfile
+import argparse
+import glob
+import timeit
```

Comment:
Which of these are additional dependencies? Do we need another requirements file?

Reply:
Nope, those are in from Python